xarray/backends/zarr.py

```diff
@@ -1218,64 +1218,85 @@ def open_datatree(
         zarr_version=None,
         **kwargs,
     ) -> DataTree:
-        from xarray.backends.api import open_dataset
         from xarray.core.datatree import DataTree
+
+        filename_or_obj = _normalize_path(filename_or_obj)
+        groups_dict = self.open_groups_as_dict(filename_or_obj, **kwargs)
+
+        return DataTree.from_dict(groups_dict)
+
+    def open_groups_as_dict(
```

Comment:
Updated this so it's more inline with the `netcdf4_.py` and `h5netcdf.py` backends. Let me know if these parameters need to be updated I copied them from `open_datatree` above.